src/Auth0.OidcClient.Android/Auth0ClientActivity.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Android.App;
+using Android.Content;
+
+namespace Auth0.OidcClient
+{
+    public class Auth0ClientActivity : Activity
+    {
+        protected override void OnResume()
+        {
+            base.OnResume();
+            ActivityMediator.Instance.Cancel();
+        }
+
+        protected override void OnNewIntent(Intent intent)
+        {
+            base.OnNewIntent(intent);
+            ActivityMediator.Instance.Send(intent.DataString);
```

Comment:
If this line resolves to parse an authentication result (either good or bad) that is where the call should end, right? No need to pass down the intent with `base.OnNewIntent(intent)`. Unless.. you can't know if the intent received here is a result of an authentication. In that case, keep both lines.

Reply:
Yeah I don't want to bypass any of their own custom onnewintent code.